ngraph/frontend/paddlepaddle/src/model.cpp

```diff
@@ -368,6 +446,7 @@ namespace ngraph
         void InputModelPDPD::InputModelPDPDImpl::overrideAllInputs(
             const std::vector<Place::Ptr>& inputs)
         {
+            m_graph_changed = true;
```

Comment:
why `determine_cut_nodes` is not called here, but delayed?

Reply:
No need to do it here. It is only needed when conversion started.